tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -178,6 +203,34 @@ def test_ops_mpc_public(get_clients, nr_clients, op_str) -> None:
     assert np.allclose(result, expected_result, atol=10e-4)
 
 
+def test_ops_divfloat_exception() -> None:
```

Comment:
MAYBE: There is a `get_clients` fixture in `conftest.py`. Can't you use that for clients?

Reply:
Oh, I can. I will make the change.